packages/backend/src/database/migrations/20230905070644_add_default_user_attribute.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable('user_attributes', (t) => {
+        t.text('default').nullable();
```

Comment:
might be safer to rename this since `default` is a reserved word in sql. Maybe `attribute_default` or `default_attribute_value`